tests/timeseries/_battery_pool/test_battery_pool.py

```diff
@@ -1101,50 +1052,22 @@ async def run_power_bounds_test(  # pylint: disable=too-many-locals
         ),
     )
 
-    # One battery stopped sending data, inverter data should be used.
-    await streamer.stop_streaming(batteries_in_pool[1])
-    await asyncio.sleep(MAX_BATTERY_DATA_AGE_SEC + 0.2)
-    msg = await asyncio.wait_for(receiver.receive(), timeout=waiting_time_sec)
-    compare_messages(
-        msg,
-        PowerMetrics(
-            now,
-            Bounds(Power.from_watts(-500), Power.from_watts(450)),
-            Bounds(Power.from_watts(-180), Power.from_watts(180)),
-        ),
-        0.2,
-    )
-
-    # All batteries stopped sending data, use inverters only.
-    await streamer.stop_streaming(batteries_in_pool[0])
-    await asyncio.sleep(MAX_BATTERY_DATA_AGE_SEC + 0.2)
-    msg = await asyncio.wait_for(receiver.receive(), timeout=waiting_time_sec)
-    compare_messages(
-        msg,
-        PowerMetrics(
-            now,
-            Bounds(Power.from_watts(-600), Power.from_watts(450)),
-            Bounds(Power.from_watts(-180), Power.from_watts(180)),
-        ),
-        0.2,
-    )
-
```

Comment:
I don't really understand why this is no longer needed :angel: Can you please clarify it?

Reply:
The behavior was changed so that if we don't have batteries, we don't use the inverters as backup to get the power-bounds. Previously with a simple 1:1 relation that was easier and more straight forward to do, with more than one inverter per battery that logic won't work anymore. Additionally, after talks with tiyash, we figured it's not a very safe way to rely on inverter bounds only when the batteries are "temporary" not available

Reply:
Thanks for the explanation and feel free to add that useful information in the commit message